data/entities.js

```diff
@@ -11652,4 +11652,400 @@ module.exports = [
+  },
+  {
+    name: 'Dentsu',
+    homepage: 'https://www.dentsuaegisnetwork.de/',
+    categories: ['cmp'],
+    domains: ['dan.mgr.consensu.org'],
+  },
+  {
+    name: 'Didomi',
+    homepage: 'https://www.didomi.io/en/',
+    categories: ['cmp'],
+    domains: ['didomi.mgr.consensu.org'],
+  },
+  {
+    name: 'Ensighten',
+    homepage: 'https://www.ensighten.com/',
+    categories: ['cmp'],
+    domains: ['ensighten.mgr.consensu.org'],
+  },
+  {
+    name: 'Evidon',
+    homepage: 'https://evidon.com',
+    categories: ['cmp'],
+    domains: ['evidon.mgr.consensu.org'],
+  },
+  {
+    name: 'Ezoic',
+    homepage: 'https://www.ezoic.com/',
+    categories: ['cmp'],
+    domains: ['ezoic.mgr.consensu.org'],
+  },
+  {
+    name: 'Gemius',
+    homepage: 'https://www.gemius.com',
+    categories: ['cmp'],
+    domains: ['gemius.mgr.consensu.org'],
+  },
+  {
+    name: 'NitroPay',
+    homepage: 'https://nitropay.com/',
+    categories: ['cmp'],
+    domains: ['nitropay.mgr.consensu.org'],
+  },
+  {
+    name: 'Google',
+    homepage: 'https://fundingchoices.google.com/start/',
+    categories: ['cmp'],
+    domains: ['fundingChoices.mgr.consensu.org'],
+  },
+  {
+    name: 'Gravito',
+    homepage: 'https://www.gravito.net/',
+    categories: ['cmp'],
+    domains: ['gravito.mgr.consensu.org'],
+  },
+  {
+    name: 'ID Ward',
+    homepage: 'https://id-ward.com/enterprise',
+    categories: ['cmp'],
+    domains: ['idward.mgr.consensu.org'],
+  },
+  {
+    name: 'iubenda',
+    homepage: 'https://www.iubenda.com',
+    categories: ['cmp'],
+    domains: ['iubenda.mgr.consensu.org'],
+  },
+  {
+    name: 'Jump',
+    homepage: 'https://jumpgroup.it/',
+    categories: ['cmp'],
+    domains: ['avacy.mgr.consensu.org'],
+  },
+  {
+    name: 'LiveRamp',
+    homepage: 'https://liveramp.com/',
+    categories: ['cmp'],
+    domains: ['faktor.mgr.consensu.org'],
+  },
+  {
+    name: 'Madvertise',
+    homepage: 'https://madvertise.com/en/',
+    categories: ['cmp'],
+    domains: ['madvertise.mgr.consensu.org'],
+  },
+  {
+    name: 'Mairdumont Netletic',
+    homepage: 'https://www.mairdumont-netletix.com/',
+    categories: ['cmp'],
+    domains: ['mdnxmp.mgr.consensu.org'],
+  },
+  {
+    name: 'Marfeel',
+    homepage: 'https://www.marfeel.com/',
+    categories: ['cmp'],
+    domains: ['marfeel.mgr.consensu.org'],
+  },
+  {
+    name: 'Mediavine',
+    homepage: 'https://www.mediavine.com/',
+    categories: ['cmp'],
+    domains: ['mediavine.mgr.consensu.org'],
+  },
+  {
+    name: 'ConsentServe',
+    homepage: 'https://www.consentserve.com/',
+    categories: ['cmp'],
+    domains: ['consentserve.mgr.consensu.org'],
+  },
+  {
+    name: 'Next14',
+    homepage: 'https://www.next14.com/',
+    categories: ['cmp'],
+    domains: ['next14.mgr.consensu.org'],
+  },
+  {
+    name: 'AdRoll',
+    homepage: 'https://www.adroll.com/',
+    categories: ['cmp'],
+    domains: ['adroll.mgr.consensu.org'],
```

Comment:
is `d.adroll.mgr.consensu.org` not valid? more generally, should we be concerned about subdomains of these being attributed?

Reply:
Just queried the HTTPArchive data and it looks like some CMPs are using deeper subdomains and some aren't, so yes I think we should also add `*.` e.g `*.adroll.mgr.consensu.org` subdomains for attribution too.